data/json/items/armor/shields.json

```
@@ -161,5 +161,72 @@
     "bashing": 10,
     "encumbrance": 30,
     "techniques": [ "WBLOCK_3" ]
+  },
+  {
+    "id": "shield_welded",
+    "type": "ARMOR",
+    "name": { "str": "welded shield" },
+    "description": "A shield made out of welded sheet metal.  It's a lot heavier and less well-made compared to a proper medieval shield, but it certainly works.",
+    "weight": "8000 g",
+    "volume": "5 L",
```

Steel plate is a bit over 12 kg, so rounded it down to 12.
```suggestion
    "description": "A shield made out of steel armor plating with handles welded on.  It's a lot heavier and less well-made compared to a proper medieval shield, but it certainly works.",
    "weight": "12 kg",
    "volume": "3 L",
```